NC-2703: CAPTCHA -> Interactive Challenge (#8157)
cloudflare/cloudflare-docs: content/fundamentals/get-started/concepts/cloudflare-challenges.md

### Interactive Challenge

Interactive challenges require a visitor to interact with the challenge page, presenting the visitor with an interactive challenge to solve. Cloudflare does not recommend using Interactive Challenges.

> For more on why Cloudflare does not recommend using Interactive Challenge, in favor of Managed Challenge, refer to our [blog](https://blog.cloudflare.com/end-cloudflare-captcha/).

Reviewer: This may not be as relevant to link any more, as `Interactive Challenge` was really a cumbersome CAPTCHA.

Reviewer: Agree, but that's the line above, not this one?

Reviewer: The previous paragraph already says `Cloudflare does not recommend using Interactive Challenges.`

Author: i think this is fine as is.

---

> ## Resolve a Interactive Challenge

Reviewer: ```suggestion
## Resolve a challenge
```

> If a visitor encounters an interactive challenge, Cloudflare employees cannot remove that challenge. Only the website owner can configure their Cloudflare settings to stop the interactive challenge to be presented.

Reviewer: ```suggestion
If a visitor encounters a challenge, Cloudflare employees cannot remove that challenge. Only the website owner can configure their Cloudflare settings to stop the challenge being presented.
```

> When observing a Cloudflare Challenge page page, a visitor could:

Reviewer 2: ```suggestion
When observing a Cloudflare Challenge page, a visitor could:
```

- Successfully pass the challenge to visit the website. Cookies and JavaScript support are required in browser settings to pass the challenge.
- Request the website owner to allow their IP address.
- Scan their computer for malicious programs (it may be infected).

> - Check their antivirus or firewall service to make sure it is not blocking access to the challenge resources (e.g. images).

Reviewer 2: ```suggestion
- Check their antivirus or firewall service to make sure it is not blocking access to the challenge resources (for example, images).
```
